src/core/debug/examples.js

```diff
@@ -155,11 +156,39 @@ const exampleScreens = {
             back: "debug",
         },
     },
+    "shop-demo": {
+        scene: ShopDemo,
+        title: "Shop Demo Game",
+        routes: {
+            back: "debug",
+            game: "debug-shop-demo-game",
+            shop: "debug-shop-demo-shop",
+        },
+    },
+    "shop-demo-game": {
+        scene: ShopDemoGame,
+        title: "Shop Demo Game",
+        routes: {
+            back: "debug-shop-demo",
+        },
+        // settings: {
```

Comment:
This is how we usually enable physics per-scene, but doesn't work when the scene is called from debug.